<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[Оператор] ЗНТУ-Абітурієнт</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="registration">
       <attribute name="title">
        <string>Реєстрація</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QPushButton" name="addBtn">
          <property name="text">
           <string>Додати</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Контактний телефон (380)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Домашня адреса</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Освітньо-кваліфікаційний рівень</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Спеціальність</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Форма навчання</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Ідентифікаційний номер</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Рейтинговий бал</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>По-батькові</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Прізвище</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="idField"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Ім'я</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Паспорт</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Потребує гуртожитку</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="passportField"/>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="telephoneField">
            <property name="text">
             <string>380</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QComboBox" name="levelBox">
            <item>
             <property name="text">
              <string>Бакалавр</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Магістр</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLineEdit" name="adressField"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="fathersNameField"/>
          </item>
          <item row="15" column="2">
           <widget class="QCheckBox" name="chummeryCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QComboBox" name="specialityBox">
            <item>
             <property name="text">
              <string>Банківська справа</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Соціальна робота</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Політологія</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Телекомунікації</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Менеджмент</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Комп. науки</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Журналістика</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Інженерія програмного забезпечення</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QComboBox" name="formBox">
            <item>
             <property name="text">
              <string>Денна</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Заочна</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="14" column="2">
           <widget class="QCheckBox" name="outsideCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QDoubleSpinBox" name="ratingBox">
            <property name="minimum">
             <double>100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>210.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Дата народження</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QDateEdit" name="dateOfBirth"/>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Поза конкурсом</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="surnameField"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="nameField"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="list">
       <attribute name="title">
        <string>Списки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QGroupBox" name="findCondition">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="1" column="0">
                <widget class="QCheckBox" name="specialityCheckBox">
                 <property name="text">
                  <string>Спеціальність</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QCheckBox" name="surnameCheckBox">
                 <property name="text">
                  <string>Прізвище</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="surnameFindField">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="specialityFindBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDateTimeEdit" name="dateFindBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="displayFormat">
                  <string>dd.MM.yyyy</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="dateCheckBox">
                 <property name="text">
                  <string>За датою додавання</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="findBtn">
               <property name="text">
                <string>Знайти</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="refreshBtn">
               <property name="text">
                <string>Скинути</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="abitList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Слід обрати БД для завантаження в меню &quot;Файл&quot;.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="loadFromDB"/>
    <addaction name="generateXML"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="loadFromDB">
   <property name="text">
    <string>Завантажити з БД</string>
   </property>
  </action>
  <action name="generateXML">
   <property name="text">
    <string>Згенерувати XML</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>nameField</tabstop>
  <tabstop>fathersNameField</tabstop>
  <tabstop>dateOfBirth</tabstop>
  <tabstop>passportField</tabstop>
  <tabstop>idField</tabstop>
  <tabstop>telephoneField</tabstop>
  <tabstop>adressField</tabstop>
  <tabstop>levelBox</tabstop>
  <tabstop>ratingBox</tabstop>
  <tabstop>formBox</tabstop>
  <tabstop>specialityBox</tabstop>
  <tabstop>outsideCheckBox</tabstop>
  <tabstop>chummeryCheckBox</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>surnameCheckBox</tabstop>
  <tabstop>surnameFindField</tabstop>
  <tabstop>specialityFindBox</tabstop>
  <tabstop>dateFindBox</tabstop>
  <tabstop>dateCheckBox</tabstop>
  <tabstop>specialityCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
